src/components/Calculator.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import './Calculator.css';
+import PropTypes from 'prop-types';
+
+class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    const { updateDetails, value } = this.props;
+    return (
+      <div className="container">
+        <div className="screen">
+          {value || 0}
+        </div>
+        <div className="buttons">
+          <button onClick={updateDetails} type="button">AC</button>
+          <button onClick={updateDetails} type="button">+/-</button>
+          <button onClick={updateDetails} type="button">%</button>
+          <button onClick={updateDetails} className="orange" type="button">&divide;</button>
+          <button onClick={updateDetails} type="button">7</button>
+          <button onClick={updateDetails} type="button">8</button>
+          <button onClick={updateDetails} type="button">9</button>
+          <button onClick={updateDetails} className="orange" type="button">&times;</button>
+          <button onClick={updateDetails} type="button">4</button>
```

Comment:
- [ ] according to the requirements the `Calculator` component should be refactored to a functional component, which might look something like the below:
```
import React from 'react';

const Calculator = () => {
  return (
    <div>

    </div>
  )
}

export default Calculator
```